tests/ctst/common/common.ts

```diff
@@ -307,6 +308,9 @@ Then('kafka consumed messages should not take too much place on disk', { timeout
 
             // If a topic remains in this array, it means it has not been cleaned
             assert(topics.length === 0, `Topics ${topics.join(', ')} still have not been cleaned`);
+        } catch (err: unknown) {
```

Comment:
is this not done in cucumber/ctst?
should it not be?

Reply:
We have a finally that clears the timeout even if we got an error, which mean we never stop the test as its timeout was -1

Reply:
Example: https://github.com/scality/Zenko/actions/runs/10668356308

Reply:
what I mean is that (even with the catch) the exception should be propagated: and eventually reach cucumber/ctst where it shoudl be caught.... and immediately fail the test (even with no timeout)

Reply:
Maybe the timeout was not due to an error... At least this ensures we log the error with a message

Errors not catched by the tests are sent to cucumber directly, it'll mark the test as failed. But in the build above, it didn't work